src/SFA.DAS.AANHub.Database/Tables/Audit.sql

```diff
@@ -2,7 +2,7 @@
 (
     [Id] BIGINT NOT NULL IDENTITY(1, 1) PRIMARY KEY, 
     [AuditTime] DATETIME2 NOT NULL, 
-    [ActionedBy] UNIQUEIDENTIFIER NOT NULL, 
+    [ActionedBy] NVARCHAR(256) NOT NULL, 
```

Comment:
Changed as errors were thrown when auditing with same memberId

Reply:
There should not be any constraint errors as we have no foreign key to member, so what was it?

Reply:
Apologies, this is a misnomer. The error that I saw was unrelated to this. `uniqueidenifier` has been re-instated.